source/features/open-all-notifications.tsx

```diff
@@ -84,7 +85,9 @@ function addOpenUnreadButtons(): void {
 
 function init(): Deinit {
 	const deinit = [delegate(document, '.' + openSelectedButtonClass, 'click', openSelectedNotifications)];
-	select(notificationHeaderSelector + ' .js-notifications-mark-selected-actions')!.append(
+	appendBefore(
+		notificationHeaderSelector + ' .js-notifications-mark-selected-actions',
+		'details',
```

Comment:
I've used `appendBefore` to avoid race conditions once #5704 is implemented.

![grafik](https://user-images.githubusercontent.com/202916/174972950-4b9061c7-c8b5-4c22-a2ef-bf83f45a108c.png)